samples/javascript/package-lock.json

```diff
@@ -552,25 +575,20 @@
             }
         },
         "follow-redirects": {
-            "version": "1.7.0",
-            "resolved": "https://registry.npmjs.org/follow-redirects/-/follow-redirects-1.7.0.tgz",
-            "integrity": "sha512-m/pZQy4Gj287eNy94nivy5wchN3Kp+Q5WgUPNy5lJSZ3sgkVKSYV/ZChMAQVIgx1SqfZ2zBZtPA2YlXIWxxJOQ==",
+            "version": "1.5.10",
```

Comment:
Are we intentionally moving backwards on this dependency?

Reply:
Yes; that's the result of upgrading our axios dependency from 0.18.0 to 0.19.0. They dropped to follow-redirects 1.5.0 between those releases.